specification/metadata/schema/schemaid.md

```diff
@@ -12,6 +12,15 @@ SchemaId
 
 Schema ID
 
+## Content constraints
+
+| Constraint    | Value                 |
+|:--------------|:----------------------|
+| Feature level | Mandatory             |
+| Allows nulls  | False                 |
+| Data type     | STRING                |
+| Value format  | Recommend GUID String |
```

Comment:
SchemaId recommended `UUID`, but we have `GUID` here.  Should we align?

Reply:
Good call will fix.

Reply:
Fixed.